tests/level1_test.go

```diff
@@ -1,32 +1,57 @@
+// +build level1
+
 package caf_tests
 
 import (
-	"context"
 	"testing"
 
-	"github.com/gruntwork-io/terratest/modules/azure"
 	"github.com/stretchr/testify/assert"
 )
 
-func TestFoundationResourceGroupsDoesNotExist(t *testing.T) {
+func TestFoundationsLandingZoneKey(t *testing.T) {
+	//arrange
 	t.Parallel()
+	tfState := NewTerraformState(t, "caf_foundations")
+
+	//act
+	landingZoneKey := tfState.GetLandingZoneKey()
 
-	test := prepareTestTable()
+	//assert
+	assert.Equal(t, "caf_foundations", landingZoneKey)
+}
 
-	client, _ := azure.GetResourceGroupClientE(test.SubscriptionID)
+func TestFoundationClientConfigSubscriptionId(t *testing.T) {
+	//arrange
+	t.Parallel()
+	tfState := NewTerraformState(t, "caf_foundations")
 
-	result, _ := client.List(context.Background(), "tagName eq 'level' and tagValue eq 'level1'", nil)
+	//act
+	client_config := tfState.GetClientConfig()
 
-	rgList := result.Values()
+	//assert
+	assert.Equal(t, tfState.SubscriptionID, client_config["subscription_id"].(string))
+}
 
-	actual := 0
-	for _, rg := range rgList {
-		if *rg.Tags["environment"] == test.Environment {
-			actual++
-		}
-	}
+func TestFoundationGlobalSettingsEnvironment(t *testing.T) {
+	//arrange
+	t.Parallel()
+	tfState := NewTerraformState(t, "caf_foundations")
+
+	//act
+	global_settings := tfState.GetGlobalSettings()
+
+	//assert
+	assert.Equal(t, tfState.Environment, global_settings["environment"].(string))
+}
+
+func TestFoundationGlobalSettingsEnvironment(t *testing.T) {
```

Comment:
seems like a duplicate of same test on line 35 ?

Reply:
It Is, copy paste error. I removed one of the duplicates.